src/app/analyzer-table/analyzer-table.component.ts

```diff
@@ -83,6 +86,19 @@ export class AnalyzerTableComponent implements OnInit, OnChanges, AfterViewCheck
     return this._table.checkTableWidth(this.tableWidths);
   }
 
+  checkSummaryStats() {
+    let i = 0, missing;
+    while (i < this.series.length) {
+      if (this.series[i].summaryStats.missing) {
+        missing = true;
+        break;
+      }
+      missing = false;
+      i++;
+    }
+    return missing;
+  }
```

Comment:
The `for` loop is better than `while` when you are looping through an array. Even prettier in this case is `some`: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some
Also a function that returns a boolean should be named informatively, e.g., `isSummaryStatMissing()`
```
isSummaryStatMissing() {
  return this.series.some((s) => s.summaryStats.missing);
}
```